lib/sqlalchemy/sql/ddl.py

```diff
@@ -434,12 +435,8 @@ class _CreateDropBase(ExecutableDDLElement):
     def __init__(
         self,
         element,
-        if_exists=False,
```

Comment:
**mike bayer** (zzzeek) wrote:

well this was just trying to have the state needed for creates/drops, but I get why you changed that here.  So I think we probably should make `_Create(_CreateDropBase)` and `_Drop(_CreateDropBase)` so that the if_exists / if_not_exists is present for subclasses already.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4067